EIPS/eip-4494.md

```diff
@@ -0,0 +1,207 @@
+---
+eip: 4494
+title: Permit for ERC721 NFTs
+description: ERC712-singed approvals for ERC721 NFTs
+author: Simon Fremaux (@dievardump), William Schwab (@wschwab)
+discussions-to: https://ethereum-magicians.org/t/eip-extending-erc2612-style-permits-to-erc721-nfts/7519/2
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-11-25
+requires: 165, 712, 721
+---
+
+## Abstract
+The "Permit" approval flow outlined in [EIP-2612](./eip-2612.md) has proven a very valuable advancement in UX by creating gasless approvals for ERC20 tokens. This EIP extends the pattern to ERC721 NFTs. This EIP borrows heavily from EIP-2612.
+
+This requires a separate EIP due to the difference in structure between ERC20 and ERC721 tokens. While ERC20 permits use value (the amount of the ERC20 token being approved) and a nonce based on the owner's address, ERC721 permits focus on the `tokenId` of the NFT and increment nonce based on the transfers of the NFT.
+
+## Motivation
+The permit structure outlined in [EIP-2612](./eip-2612.md) allows for a signed message (structured as outlined in [EIP-712](./eip-712.md)) to be used in order to create an approval. Whereas the normal approval-based pull flow generally involves two transactions, one to approve a contract and a second for the contract to pull the asset, which is poor UX and often confuses new users, a permit-style flow only requires signing a message and a transaction. Additional information can be found in [EIP-2612](./eip-2612.md).
```

Comment:
```suggestion
The permit structure outlined in [EIP-2612](./eip-2612.md) allows for a signed message (structured as outlined in [ERC-712](./eip-712.md)) to be used in order to create an approval. Whereas the normal approval-based pull flow generally involves two transactions, one to approve a contract and a second for the contract to pull the asset, which is poor UX and often confuses new users, a permit-style flow only requires signing a message and a transaction. Additional information can be found in [EIP-2612](./eip-2612.md).
```

Reply:
When should standards be referred to as an EIP and when as an ERC?

(I went for colloquial references being ERC, but actual references to standards being EIP)